src/oncall/api/v0/user.py

```diff
@@ -143,7 +143,11 @@ def on_put(req, resp, user_name):
     cursor = connection.cursor()
     if set_clause:
         query = 'UPDATE `user` SET {0} WHERE `name` = %s'.format(set_clause)
-        query_data = tuple(data[field] for field in data) + (user_name,)
+        query_data = ()
```

Comment:
Can we do a list here with appends instead of +=? Should be slightly more efficient. Also, please add a test in e2e/test_users.py for updating contacts.